rust/parquet/src/util/cursor.rs

```diff
@@ -0,0 +1,203 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I discovered above that the entire `util` module is not public -- 

https://github.com/apache/arrow/blob/master/rust/parquet/src/lib.rs#L39

```
#[macro_use]
mod util;
```

So none of these pub traits can be used outside this crate at the moment

Reply:
this is intended